### PR TITLE
Update caption on beardofedu file

### LINKED ISSUE #10: beardofedu wants to learn...
- [ ] github flow 
- [ ] what are actions?
- [ ] dealing with merge conflicts! 

- 👋

### PR DESCRIPTION
**Pull request by beardofedu**
This updates the caption of the beardofedu slide. 

Closes #10

### CODE REVIEW

File: _slides/02-beardofedu.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+this image makes me want to try that new ps4 game :thinking:
```
Comment (alecreveles-isd) [inline _slides/02-beardofedu.md:6]: Ghost of Tsushima?

File: _slides/02-beardofedu.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (alecreveles-isd) [inline _slides/02-beardofedu.md:3]:
```suggestion
title: "Welcome to beardofedu's slide!"
```

File: _slides/02-beardofedu.md (hunk 1 above)
Comment (alecreveles-isd) [inline _slides/02-beardofedu.md:6]: Ghost of Tsushima?

Comment (beardofedu) [reply _slides/02-beardofedu.md:6]: Yea!

**Pull request merged**